src/locales/en/translation.json

```
@@ -243,14 +251,199 @@
   "go_to_explore_sample_event": "View this sample event on MERMAID Explore",
   "go_to_explore_this_project": "View this project on MERMAID Explore",
   "gfcr": {
-    "title": "Global Fund for Coral Reefs (GFCR)",
+    "gfcr": "GFCR",
+    "gfcr_long": "Global Fund for Coral Reefs (GFCR)",
     "add_gfcr_indicators": "GFCR is a global partnership that aims to mobilize resources to support coral reef conservation and restoration projects around the world.",
     "remove_gfcr_Indicators": "Removing GFCR indicators from this project will not delete them, but just hide them.",
     "disabled_gfcr_info": "Disabling GFCR Indicators for this project will not delete them, but just hide them from the <a />. No data will be lost. You can re-enable them at any time.",
+    "create_new": "Create new",
+    "exporting": "Exporting...",
+    "report": "Report",
+    "target": "Target",
+    "untitled": "Untitled",
+    "no_indicator_sets": "No indicator sets yet.",
+    "no_indicator_info": "Select 'Create new' to add an indicator set to this project.",
+    "indicator_set": "Indicator Set",
+    "create_indicator_set": "Create Indicator Set",
+    "reporting_date": "Reporting date",
+    "reporting_date_year": "Reporting Date Year",
+    "indicator_set_title_info": "Recommended naming conventions: For target reports, use \"Phase 1 Target\", \"Mid-term Target\", or \"Final Target\". For bi-annual progress reports, use \"Mid-year Report\" or \"End-year Report\".",
+    "indicator_set_date_info": "The date for which this indicator set is being reported.",
+    "hide_help": "Hide Help",
+    "show_help": "Show Help",
+    "fund_indicators": "FUND INDICATORS",
+    "report_title_and_date": "Title and date",
+    "programme_area": "Programme area",
+    "conservation_and_management": "Conservation and management",
+    "restoration": "Restoration",
+    "coral_reef_health": "Coral reef health",
+    "communities": "Communities",
+    "people": "People",
+    "climate_response": "Climate response",
+    "businesses_finance_solutions": "Businesses / Finance solutions",
+    "investments": "Investments",
+    "revenues": "Revenues",
+    "errors": {
+      "indicator_sets_unavailable": "GFCR indicator sets are currently unavailable.",
+      "indicator_set_save_failed": "There was an error saving the indicator set.",
+      "form_errors": "Form errors:"
+    },
+    "success": {
+      "indicator_set_save": "Indicator set saved successfully."
+    },
     "buttons": {
       "go_to_gfcr": "Go to GFCR Indicators",
       "disable_gfcr": "Disable GFCR Indicators",
       "enable_gfcr": "Enable GFCR Indicators for this project"
+    },
+    "forms": {
+      "f1_heading": "Coral reef extent of GFCR Programme",
```

The numbers don't add to the context and go against our naming standards. Can the context of these key values be summarized as the key name?
```suggestion
      "gfcr_reef_extent": "Coral reef extent of GFCR Programme",
```

---

Let's also move this up a namespace so it's not so deeply nested. 'forms' doesn't seem to add much context here.

---

Yeah - the `F#` doesn't add to translation or verbiage contextually, but the actual verbiage could.